src/pxl_scripts/bpftrace/socket_size/data.pxl

```diff
@@ -0,0 +1,177 @@
+# Adapted From "BPF Performance Tools" book by Brendan Gregg; Chapter 10: Networking
+
+import pxtrace
+import px
+
+sendmsg_program = """
+// socketsize tool shows the socket stats (count, size, and throughput) for both read and write socket I/O requests.
+
```

Comment:
Please include the license from https://github.com/brendangregg/bpf-perf-tools-book/blob/master/originals/Ch10_Networking/socksize.bt#L1-L14 here.
Perhaps also a link to the source.